test/testRoute.js

```diff
@@ -0,0 +1,102 @@
+'use strict'
+let chai = require('chai');
+let assert = chai.assert;
+let should = chai.should();
+let Hapi = require('hapi');
+let request = require('request');
+
+let launchServer = function(server, port, options, done){
+  // server = new Hapi.Server();
+  server.connection({ port: port });
+  server.register([{
+    register: require('../'),
+    options :  options,
+  }
+  ], function (err) {
+    if (err) {
+      console.error('Failed to load plugin:', err);
+      return done(err);
+    }
+    server.start(function(err){
+      if (err){
+        console.log(err)
+        return done(err);
+      }
+      done();
+    });
+  });
+}
+
+
+describe('hapi-route-loader /dashboard base', function(done){
+  let server = new Hapi.Server();
+  let port = 8080;
+  let options = {base:'/dashboard'};
+  before(function(done){
+    launchServer(server,port,options,done);
+  });
+  it("base: '/dashboard', path: 'get' => '/dashboard/get'", function(done){
+    request.get('http://localhost:8080/dashboard/get', function(err,response){
```

Comment:
you can use server.inject also and it'll go through the hapi stack.  might make this a little cleaner